src/config.rs

```diff
@@ -1034,6 +1040,9 @@ pub struct Font {
 
     /// Extra spacing per character
     offset: FontOffset,
+
+    #[serde(default)]
```

Comment:
This should probably default to `true`. Since we've already got a helper for it, you could simply change the attribute to

```rust
#[serde(default="true_bool")] 
```